apps/admin/src/providers/AppProvider.tsx

```diff
@@ -42,8 +40,8 @@ const AppProvider: FC = () => {
     <OrderCloudProvider
       baseApiUrl={BASE_API_URL}
       clientId={CLIENT_ID}
-      scope={SCOPE}
-      customScope={CUSTOM_SCOPE}
+      scope={[]}
+      customScope={[]}
```

Comment:
Why are we removing the environment variables and connection to the provider for SCOPE and CUSTOM_SCOPE?

Reply:
Is this because when you login with an empty scope it just provides you with all available scopes?